composer.lock

```diff
@@ -8,25 +8,25 @@
     "packages": [
         {
             "name": "brick/math",
-            "version": "0.12.3",
+            "version": "0.14.0",
```

Comment:
@Nima8FT Hello again!

These edits are outside the scope of this issue. This is a dependency update. Please revert the changes in this branch (you can simply check out the file in branch `main` using the command `git checkout main composer.lock`).